test/test.js

```diff
@@ -227,7 +227,7 @@ describe('S3rver Tests', function () {
         if (err) {
           return done(err);
         }
-        object.ContentLength.should.equal(stats.size.toString());
+        object.ContentLength.should.equal(stats.size);
```

Comment:
see: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getObject-property

The examples show numbers instead of strings